### DISCUSSION TITLE
谷歌快速付费后不能授权使用专业版，使用邮箱授权 没有邮件

### DISCUSSION THREAD

**Discussion by silverAnswerer** [getlantern/lantern-forum-cn#959] [Q&A]
谷歌快速付费后不能授权使用专业版，使用邮箱授权 没有邮件

邮箱335196774@qq.com
![Uploading Screenshot_2024-10-13-12-01-36-826_com.chinamworld.main.jpg…]()

**Comment by MaxMKMax** [comment]
@silverAnswerer 您好. 请您重新提供我支付单据. 被Github屏蔽了.

**Reply by silverAnswerer** [reply]
![b79e369f-1eb6-489a-9e1d-a0e221cd2419](https://github.com/user-attachments/assets/6e5113f4-0e41-4c17-be14-4eb9fb463cae)
![Screenshot_2024-10-15-23-42-12-277_com chinamworl](https://github.com/user-attachments/assets/7a14a200-d36f-4d2f-b056-b3727225e022)

**Reply by silverAnswerer** [reply]
感谢你的帮助，这是我的支付凭证，支付方式 Google play 商店支付 ，支付途径中国建设银行，然后联系了支持人员给我发了一份一个月的激活码，但是好像还是上不去 或者说不稳定
